src/pages/chat/class/classSearch/classbox.styles.ts

```diff
@@ -0,0 +1,37 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { pxToRem } from "~/utils";
+import { flex } from "~/styles/utils/flex";
+
+const sharedTextStyles = css`
+  font-family: "NanumSquare";
+  font-weight: 600;
+`;
+
+export const ClassBox = styled.div`
+  height: 85px;
+  width: 100%;
+  display: flex;
+  ${flex({ justifyContent: "space-between", alignItems: "center" })}
+  border-bottom: ${pxToRem(0.6)} solid #707070;
+  padding: 0 ${pxToRem(12)} 0 ${pxToRem(12)};
```

Comment:
```suggestion
  padding: 0 ${pxToRem(12)};
```
으로 적을 수 있습니다.
불필요하게 가독성이 떨어지는 작성 방법 같습니다 ㅠ